posts/Asmaa-intro/index.qmd

```diff
@@ -1,11 +1,41 @@
 ---
-title: ""
-author: "Asmaa Kasim"
+title: "Outreachy - Introduce yourself"
+author: "Asma Kacem"
 date: "2022-05-30"
 categories: [outreachy]
-draft: true
+
```

Comment:
subtitle: "A little bit about myself"
license: "CC BY"
toc-title: Table of contents
toc-location: left